### ISSUE TITLE
Make project stats DM consistent w/ stats language in Playbook

### ISSUE THREAD

**Issue by tannerwelsh** [LearnersGuild/echo#429]
tl;dr in talk w/ @prattsj today, they pointed out that the project stats DM was out-of-sync with the stats language used in the [playbook](http://playbook.learnersguild.org/Game_Manual/Stats.html).

I went a little further and made some other edits to formatting and language used in the interest of clarity and concision.

Changes needed, w/ example report shown below:
- Use `Actual Contribution`, `Expected Contribution`, and `Contribution Gap` (note capitalization, to make clear which stat they correlate to)
- Simplify the header text
- s/assessment/estimation
- Inclusion of a `+` character preceding the `Contribution Gap` number to clearly indicate a surplus contribution. Negative gap would be preceded by a `-` character.

---

Retrospective results for [#happy-tiger](#):

**Feedback from your team:**
- Jeffrey, I like working with you because you shower every day and I'd like working with you even more if you chewed with your mouth closed.
- Jeffrey, I like working with you because you have an interesting way of deconstructing problems, and I'd like working with you even more if you took more risks.
- ...

**Team evaluations of your performance:**

```
Learning Support: 70%
Culture Contribution: 84%
Team Play: 87%
```

**Time Dedication:**

```
Your hours: 42
Total team hours: 81
Team size: 3

@deadlyicon (Jared Grippe): 5
@jeffreywescott (Jeffrey Wescott): 42
@tannerwelsh (Tanner Welsh): 34
```

**Project Contribution:**

```
Self estimation: 40%
Team estimation: 45%

Actual Contribution to the project: 42%
Expected Contribution for # of hours: 33%
Contribution Gap: +9%
```
